locations/spiders/tofs_gb.py

```diff
@@ -0,0 +1,24 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+
+
+class TofsGBSpider(Spider):
+    name = "tofs_gb"
+    item_attributes = {
+        "brand": "The Original Factory Shop",
+        "brand_wikidata": "Q7755366",
+    }
+    start_urls = ["https://api.storepoint.co/v1/15f4fbe5b10e3d/locations?rq"]
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def parse(self, response, **kwargs):
+        for location in response.json()["results"]["locations"]:
+            item = DictParser.parse(location)
+            item["lat"] = location["loc_lat"]
+            item["lon"] = location["loc_long"]
+            item["branch"] = item["name"]
+            item["name"] = "The Original Factory Shop"
+            item.pop("website")
+            item["facebook"] = location["facebook"]
+            yield item
```

Comment:
May I also kindly ask to hardcode category please?
If somebody adds cameras operator with this brand the pipeline will not be able to assign category automatically. 

```suggestion
            item["facebook"] = location["facebook"]
            apply_categoy(Categories.SHOP_VARIETY_STORE, item)
            yield item
```